scripts/set-more-info-link.py

```diff
@@ -6,50 +6,49 @@
 import sys
```

Comment:
```suggestion
import sys
import requests
```

Reply:
Could this not be merged into one line.
```suggestion
import sys, requests
```

Reply:
Yes, but per [PEP-8]:

> Imports should usually be on separate lines:
> ```
> # Correct:
> import os
> import sys
> ```
> ```
> # Wrong:
> import sys, os
> ```
> It's okay to say this though:
> ```
> # Correct:
> from subprocess import Popen, PIPE
> ```